pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/maler/example/DesignReferenceLetter.kt

```diff
@@ -0,0 +1,212 @@
+            // section title
+            title1 {
+                text(Bokmal to "Hvorfor skriver vi til deg?", Nynorsk to "Hvorfor skriver vi til deg?")
+            }
+
+            paragraph {
+                text(
+                    Bokmal to "Du har fått innvilget pensjon. Dette er grunnen til det. Dette er grunnen til det. Dette er grunnen til\n" +
+                            "det. Dette er grunnen til det. Dette er grunnen til det. Dette er grunnen til det. Du får 16 000 kroner i\n" +
+                            "måneden fra januar 2022.",
+                    Nynorsk to "Du har fått innvilget pensjon. Dette er grunnen til det. Dette er grunnen til det. Dette er grunnen til\n" +
+                            "det. Dette er grunnen til det. Dette er grunnen til det. Dette er grunnen til det. Du får 16 000 kroner i\n" +
+                            "måneden fra januar 2022."
+                )
+            }
+
+
+            title1 {
+                text(Bokmal to "Utbetalingsoversikt", Nynorsk to "Utbetalingsoversikt")
+            }
+
+            paragraph {
+                text(
+                    Bokmal to "Dette er din månedlige pensjonsutbetaling",
+                    Nynorsk to "Dette er din månedlige pensjonsutbetaling"
+                )
+                table(
+                    header = {
+                        column(1) { text(Bokmal to "Måned", Nynorsk to "Måned", FontType.BOLD) }
+                        column(1, RIGHT) { text(Bokmal to "Stønad", Nynorsk to "Stønad", FontType.BOLD) }
+                        column(1, RIGHT) { text(Bokmal to "Pensjon", Nynorsk to "Pensjon", FontType.BOLD) }
+                        column(1, RIGHT) { text(Bokmal to "Totalt", Nynorsk to "Totalt", FontType.BOLD) }
+                    }
+                ) {
+                    listOf(
+                        "Januar",
+                        "Februar",
+                        "Mars",
+                        "April",
+                        "Mai",
+                        "Juni",
+                        "Juli",
+                        "August",
+                        "September",
+                        "Oktober",
+                        "November",
+                        "Desember",
+                        "Desember",
+                        "Januar",
+                        "Februar",
+                        "Mars",
+                    ).forEach {
+                        row {
+                            cell { text(Bokmal to it, Nynorsk to it) }
+                            cell { text(Bokmal to "1 Kr", Nynorsk to "1 Kr") }
+                            cell { text(Bokmal to "1 Kr", Nynorsk to "1 Kr") }
+                            cell { text(Bokmal to "2 Kr", Nynorsk to "2 Kr") }
+                        }
+                    }
+                }
+            }
+
+            title1 {
+                text(Bokmal to "Tittel", Nynorsk to "Tittel")
+            }
+
+            paragraph {
```

Comment:
Kunne dette vært en frase?

Reply:
Joda, men dette er ett test brev som endres mye på for å matche prototypen i figma, så ser ikke helt poenget.